docs/topics/native/native-spm.md

```diff
@@ -0,0 +1,192 @@
+[//]: # (title: Swift Package export setup)
+
+Kotlin/Native output for an Apple target can be consumed as an SPM dependency.
+For example, if your Kotlin Multiplatform project has an iOS target, you may want to make the binary available
+as a dependency to iOS developers working on native Swift projects in Xcode. Using KMP tooling, you can provide
+an artifact that would seamlessly integrate with Xcode experience.
+
+This guide shows how to do this using the [XCFramework building](multiplatform-build-native-binaries.md#build-xcframeworks)
+capability of Kotlin Gradle.
+
+## Prepare locations for the files to be uploaded
+
+To make your framework available to be consumed, you will need to upload two files:
+* A ZIP archive of the XCFramework itself should be uploaded to a convenient file storage with direct access (for example,
```

Comment:
I think the list has too many modal verbs which makes the instructions sound very strict. You can both highlight the files and use the imperative:

> To make your framework consumable, you need to upload two files:
> 
> * A ZIP archive of the XCFramework. Upload it to a convenient file storage with direct access, for example, an S3, GitHub, or Maven repository. Choose the option that is easiest to integrate into your workflow.
> * The Package.swift file, describing the package. Prepare a Git repository and upload the file. To decide which repository to use, see the pros and cons of several options below.